frontend/src/store/app.js

```diff
@@ -127,10 +127,16 @@ export const useAppStore = defineStore('app', {
       const orderNumber = state?.programYearList.list.find(
         (el) => el.programYearId == applicationStore?.programYearId,
       )?.order;
-      if (orderNumber < 5) {
-        return PROGRAM_YEAR_LANGUAGE_TYPES.HISTORICAL;
-      } else {
-        return PROGRAM_YEAR_LANGUAGE_TYPES.FY2024_25;
+      switch (orderNumber) {
```

Comment:
question: do we need a code change to update this function every year? i.e.: what will happen when we have FY2026_27?

Reply:
For now.. yes.. it's not ideal - but we also need the program years in place in Dynamics to map them. Danna has been talking about adding 10 program years into Dynamics so perhaps we can update it all it once